backend/dal/Services/Admin/Concrete/ParcelService.cs

```diff
@@ -201,38 +205,38 @@ public IEnumerable<Parcel> Add(IEnumerable<Parcel> entities)
 
             var userId = this.User.GetUserId();
             entities.ForEach((entity) =>
-           {
-               if (entity == null) throw new ArgumentNullException();
-               entity.CreatedById = userId;
-
-               entity.Buildings.ForEach(b =>
-               {
-                   b.CreatedById = userId;
-                   this.Context.Buildings.Add(b);
-
-                   b.Evaluations.ForEach(e =>
-                   {
-                       e.CreatedById = userId;
-                       this.Context.BuildingEvaluations.Add(e);
-                   });
-
-                   if (b.Address != null)
-                   {
-                       b.Address.CreatedById = userId;
-                   }
-               });
-
-               entity.Evaluations.ForEach(e =>
-               {
-                   e.CreatedById = userId;
-                   this.Context.ParcelEvaluations.Add(e);
-               });
-
-               if (entity.Address != null)
-               {
-                   entity.Address.CreatedById = userId;
-               }
-           });
+            {
```

Comment:
I'm starting to think we want to enforce formatting on the backend as well as the frontend.